learn_django/trip_project/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import permissions
+
+
+class IsSuperUserOrStaff(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if request.user and (request.user.groups.filter(name='superuser') or
```

Comment:
why you are hardcoding the name here? Have something like, 

1. Add a proxy user model https://docs.djangoproject.com/en/3.1/topics/db/models/#proxy-models
2. Add two methods. `is_staff_member(user)` , is_su_member(user)`
3. Call them here. `User.is_staff_member()` or `User.is_su_member`
4. Impliment the two methods in the proxy model.